Scripts/RemediationScripts/Remediate-Azure_AppService_DP_Use_Secure_FTP_Deployment.ps1

```diff
@@ -0,0 +1,1132 @@
+                    catch
+                    {
+                        $nonProductionSlotsSkipped += $slot
+                        Write-Host "Error Configuring FTP State on the non-production slot." -ForegroundColor $([Constants]::MessageType.Error)
+                        Write-Host $([Constants]::SingleDashLine)
+                    }
+                }
+            }
+            
+            $nonProductionSlotsSkippedStr = $nonProductionSlotsSkipped -join ','
+            $appService.NonProductionSlotsSkipped = $nonProductionSlotsSkippedStr
+
+            if ($($nonProductionSlotsSkipped | Measure-Object).Count -eq 0)
+            {
+                $appService.isSecureFTPEnabledOnAllNonProdSlots = $true
+                $logResource = @{}
+                $logResource.Add("ResourceGroupName",($_.ResourceGroupName))
+                $logResource.Add("ResourceName",($_.ResourceName))
+                $logRemediatedResources += $logResource
+                $appServicesRemediated += $appService
+                Write-Host "Successfully Configured FTP State on production and all non-production slots for the App Service." -ForegroundColor $([Constants]::MessageType.Update)
+                Write-Host $([Constants]::SingleDashLine)
+            }
+            else
+            {
+                $appServicesSkipped += $appService
+                $logResource = @{}
+                $logResource.Add("ResourceGroupName",($_.ResourceGroupName))
+                $logResource.Add("ResourceName",($_.ResourceName))
+                $logResource.Add("Reason", "Error Configuring FTP State for these non-production slots: [$($nonProductionSlotsSkippedStr)]")
+                $logSkippedResources += $logResource
+                Write-Host "Error Configuring FTP State for these non-production slots: [$($nonProductionSlotsSkippedStr)]" -ForegroundColor $([Constants]::MessageType.Error)
+                Write-Host $([Constants]::SingleDashLine)
+                
+            }
+        }
+
+        # Write-Host $([Constants]::SingleDashLine)
+
+        if (($appServicesRemediated | Measure-Object).Count -eq $totalAppServicesWithFTPStateAllAllowed)
+        {
+            Write-Host "FTP State successfully configured on the production slot and all non-production slots for all [$($totalAppServicesWithFTPStateAllAllowed)] App Service(s)." -ForegroundColor $([Constants]::MessageType.Update)
+            
+        }
+        else
+        {
+            Write-Host "FTP State successfully configured on the production slot and all non-production slots for [$($($appServicesRemediated | Measure-Object).Count)] out of [$($totalAppServicesWithFTPStateAllAllowed)] App Service(s)." -ForegroundColor $([Constants]::MessageType.Warning)
+        }
+
+        $colsProperty = @{Expression={$_.ResourceId};Label="Resource ID";Width=40;Alignment="left"},
+                        @{Expression={$_.ResourceGroupName};Label="Resource Group Name";Width=20;Alignment="left"},
```

Comment:
Change column headers in summary to  Is Secure FTP deployment enabled on the production slot. 
Similarly for nonProd slots, lable change

Reply:
Did changes in Summary as mentioned above.